### PR TITLE
feat: add Arial as fallback for fonts

### LINKED ISSUE #1972: CLS due to fallback font
Hey folks, for the font-family used in blade, can we have fallback fonts which match the actual font very closely. 

Asking this because for pages built with blade on website, we're seeing an increased layout shift 
from fallback font to when actual font load. eg on /payment-pages/
This can we solved by matching the fallback font closely to the actual font - [ref](https://github.com/razorpay/frontend-website/pull/2894/files#r1450244048)

Below is a recording of CLS due to current font Lato and fallback BlinkMacSystemFont
created by https://meowni.ca/font-style-matcher/

https://github.com/razorpay/blade/assets/28626499/def8a76d-7ff0-4434-9651-2418c4923722

### PR DESCRIPTION
**Pull request by saurabhdaware**
## Description

Adds adjusted Arial as fallback for TASA and Inter

Resolves #1972 

## Component Checklist

- [ ] Update Component Status Page
- [ ] Perform Manual Testing in Other Browsers
- [ ] Add KitchenSink Story
- [ ] Add Interaction Tests (if applicable)
- [ ] Add changeset

### CODE REVIEW

**Review by snitin315 — COMMENTED** [review]
Are we planning to release fallback fonts for v10 too?

**Comment by saurabhdaware** [comment]
> Are we planning to release fallback fonts for v10 too?

nope

File: packages/blade/fonts.css [hunk 1]
```diff
@@ -80,3 +80,27 @@
...
+  src: local(Arial);
+  size-adjust: 98%;
+  ascent-override: normal;
+  descent-override: normal;
+  line-gap-override: 7%;
+}
+
+:root {
+  --inter-fallback: 'Inter Fallback Arial';
+  --tasa-orbiter-fallback: 'TASA Orbiter Fallback Arial';
```
Comment (anuraghazra) [inline packages/blade/fonts.css:105]: nit: prefix with blade

**Review by anuraghazra — APPROVED** [review]

**Review by snitin315 — APPROVED** [review]

**Review by snitin315 — COMMENTED** [review]
update snapshots.

**Comment by saurabhdaware** [comment]
@anuraghazra @snitin315 I removed the CSS variables implementation. I thought when CSS variables are not available, it would just fallback to next available font which was `Arial` but instead when CSS variables were not available, it was invalidating that font-family definition itself which is why chromatic snapshots were showing that many changes. It was falling back to next font-family definition.

Now I directly refer to the font-face name in code

**Review by snitin315 — APPROVED** [review]

**Pull request merged**